portfolio/src/main/webapp/index.html

```diff
@@ -129,8 +127,11 @@ <h2 id="outro">...and here we are today.</h2>
                         <h2>Previous Comments</h2>
                         <hr/>
                         <div id="msg-container"></div>
+                        <script>getComments(5)</script>
                         <hr/>
                         <input onchange="addCommentsToDOM(this.value)" type="number" id="quantity" name="limit" min="0" placeholder="Number of comments"/>
```

Comment:
Does this work? When I select '1' I don't see any change in the number of  comments on the page.